pkgs/applications/window-managers/i3/lock-color.nix

```diff
@@ -25,6 +25,9 @@ stdenv.mkDerivation rec {
   installFlags = "PREFIX=\${out} SYSCONFDIR=\${out}/etc MANDIR=\${out}/share/man";
   postInstall = ''
     mv $out/bin/i3lock $out/bin/i3lock-color
+    mv $out/etc/pam.d/i3lock $out/etc/pam.d/i3lock-color
```

Comment:
Please remove this line as it will cause problems with PAM authentication. The app starts PAM under the service name 'i3lock', so  the file must be named `/etc/pam.d/i3lock`.